enclave_manager/tcf_enclave_manager/enclave_manager.py

```diff
@@ -22,8 +22,8 @@
 import time
 import random
 
-import sgx_work_order_request as work_order_request
-import tcf_enclave_helper as enclave_helper
+import tcf_enclave_manager.sgx_work_order_request as work_order_request
```

Comment:
change directory/file names to avalon_* from tcf_*

Reply:
Will address renaming filename from tcf to avalon in another PR.

Reply:
Renaming "tcf" to "avalon" in enclave-manager context makes sense to handle that in this PR itself. There might be other such instances in other parts of the code which can be picked up later. Same way has been done in other modules of avalon also.